src/main/java/com/adobe/pdfservices/operation/samples/electronicseal/ElectronicSealWithAppearanceOptions.java

```diff
@@ -0,0 +1,125 @@
+
+            //Set the Seal Field Name to be created in input PDF document.
+            String sealFieldName = "<SEAL_FIELD_NAME>";
+
+            //Set the page number in input document for applying seal.
+            Integer sealPageNumber = 1;
+
+            //Set if seal should be visible or invisible.
+            Boolean sealVisible = true;
+
+            //Create SealFieldLocationOptions instance and set the coordinates for applying signature
+            SealFieldLocationOptions sealFieldLocationOptions = new SealFieldLocationOptions(150, 250, 350, 200);
+
+            //Create SealFieldOptions instance with required details.
+            SealFieldOptions sealFieldOptions = new SealFieldOptions.Builder(sealFieldName)
+                    .setSealFieldLocationOptions(sealFieldLocationOptions)
+                    .setPageNumber(sealPageNumber)
+                    .setVisible(sealVisible)
+                    .build();
+
+            //Set the name of TSP Provider being used.
+            String providerName = "<PROVIDER_NAME>";
+
+            //Set the access token to be used to access TSP provider hosted APIs.
+            String accessToken = "<ACCESS TOKEN>";
+
+            //Set the credential ID.
+            String credentialID = "<CREDENTIAL_ID>";
+
+            //Set the PIN generated while creating credentials.
+            String credentialPin = "<PIN>";
+
+            //Create SealCredentialOptions instance with required certificate details.
+            SealCredentialOptions sealCredentialOptions = new  CSCCredentialOptions.Builder(providerName, credentialID, credentialPin, accessToken).setTokenType("Bearer").build();
+
+            //Create SealingOptions instance with all the sealing parameters.
+            SealOptions sealOptions = new SealOptions.Builder(SignatureFormat.PKCS7, sealCredentialOptions,
+                    sealFieldOptions).setSealAppearanceOptions(sealAppearanceOptions).build();
+
+            //Create a PDFElectronicSealOptions instance using the SealOptions instance
+            PDFElectronicSealOptions pdfElectronicSealOptions = new PDFElectronicSealOptions(sealOptions);
+
+            //Create the PDFElectronicSealOperation instance using the PDFElectronicSealOptions instance
+            PDFElectronicSealOperation pdfElectronicSealOperation = PDFElectronicSealOperation.createNew(pdfElectronicSealOptions);
+
+            //Set the input source file for PDFElectronicSealOperation instance
+            pdfElectronicSealOperation.setInputDocument(sourceFile);
+
+            //Set the optional input seal image for PDFElectronicSealOperation instance
+            pdfElectronicSealOperation.setSealImage(sealImageFile);
+
+            //Execute the operation
+            FileRef result = pdfElectronicSealOperation.execute(executionContext);
+
+            //Save the output at specified location
+            result.saveAs("output/sealedOutputWithAllAppearanceOptions.pdf");
```

Comment:
Please change the output file path to "output/sealedOutputWithAppearanceOptions.pdf"